main.go

```diff
@@ -170,6 +174,9 @@ func main() {
 			args = append(args, scanner.Text())
 		}
 	}
+	if config.UseAria2RPC {
+		config.ExtractedData = true
```

Comment:
you don't have to rely on this, just add a new judgment logic in `Download `.

Reply:
Fixed these issues and rebased.